### ISSUE TITLE
Draft mode isn't working

### ISSUE THREAD

**Issue by oliver-77** [yuzoolcode/pulsecms-tracking#4]
When I set "draft-" before the file name of a blog post, the post still exists in the frontend.

![draft](https://user-images.githubusercontent.com/24696577/33794381-99afec1a-dccb-11e7-87a4-b0c1f7a6c068.png)

**Comment by IvaRoSoft** [comment]
Yes. (version 5.0)
Clicking on the link will cause an error.

```
https://ivaro.php5.sk/blog-draft_3-a-very-nice-blog-post

Fatal error: Uncaught exception 'LogicException' with message 'An invariant has failed' in /www/sites/3/site21263/public_html/pulsecore/dbc.php:12 Stack trace: #0 /www/sites/3/site21263/public_html/inc/tags/blog.php(37): pulsecore\invariant(false) #1 /www/sites/3/site21263/public_html/index.php(277): include('/www/sites/3/si...') #2 {main} thrown in /www/sites/3/site21263/public_html/pulsecore/dbc.php on line 12
```

**Comment by IvaRoSoft** [comment]
Return to admin, =
"No content found."
![blog-draft - 9_12](https://user-images.githubusercontent.com/29908185/33794801-81730a62-dcd3-11e7-961b-9565b3d479d9.png)

**Comment by oliver-77** [comment]
Is there a space between Draft- and 3? It can't be like this.

**Comment by IvaRoSoft** [comment]
Renamed via ftp. OK
Well thank you!

**Comment by ghost** [comment]
Fixed in 5.0.2! 😃👍
http://help.pulsecms.com/article/10-version-history
